Makefile

```diff
@@ -30,8 +30,17 @@ ifeq ($(ARCH),ppc64le)
 	LIB_DIR=powerpc64le-linux-gnu
 	CC=powerpc64le-linux-gnu-gcc
 endif
+ifeq ($(ARCH),s390x)
+	LIB_DIR=s390x-linux-gnu
+	CC=s390x-linux-gnu-gcc
+endif
 GOARM=6
-KUBE_CROSS_TAG=v1.6.2-2
+ifeq ($(ARCH),s390x)
+	# kube-cross:v1.7.4-1 supports s390x
+	KUBE_CROSS_TAG=v1.7.4-1
```

Comment:
KUBE_CROSS:1.7.4-1 has go which supports all the architectures including s390x(The curent version of KUBE_CROSS has go which does not support s390x). Also this new image uses glibc version 2.23.
But the busybox uses the glibc version 2.19 which is not compatible with flannel binary generated.To handle this situation for s390x, I am pulling libc.so.6 and ld64.so.1 from kube-cross to busybox.
If we upgrade the KUBE_CROSS version then either we need to change glibc version used in busybox or like s390x we will have to pull libc.so.6 and ld64.so.1 from kube-cross and use in busybox for all the architectures.